testsuite/flattening/modelica/scodeinst/WhenCondition5.mo

```diff
@@ -0,0 +1,24 @@
+// name: WhenCondition5
```

Comment:
There is no difference to `WhenCondition4.mo`

Reply:
Thanks, it was supposed to check that it worked with algorithms too but seems I forgot to change the test case. Fixed in #7016.